### PR TITLE
[5.2] [Guided Tours] Global Configuration delay field fix

### PR DESCRIPTION
**Pull request by obuisard**
### Summary of Changes

The 'delay' parameter is not filtered, allowing any character in the field.

### Testing Instructions

Go to the Global Configuration of the Guided Tours.

### Actual result BEFORE applying this Pull Request

Enter a value in the field 'Auto Start Time Delay (in minutes)'. Try entering letters.
You can set the field to any character other than numbers.

### Expected result AFTER applying this Pull Request

You can no longer enter anything else but numbers.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 78b97139f37442aadc4d12560385079f8aa662d2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44091">issues.joomla.org/tracker/joomla-cms/44091</a>.</sub>

File: administrator/components/com_guidedtours/config.xml [hunk 1]
```diff
@@ -17,11 +17,10 @@
 
 		<field
 			name="delayed_time"
-			type="text"
+			type="number"
 			label="COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_LABEL"
 			description="COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_DESCRIPTION"
 			default="60"
```
Comment (Quy) [inline administrator/components/com_guidedtours/config.xml:23]:
```suggestion
			default="60"
			min="1"
			filter="integer"
			validate="number"
```

Comment (obuisard) [reply administrator/components/com_guidedtours/config.xml:23]: I did not think the filter was necessary since the 'number' type already filters the data.

Comment (Quy) [reply administrator/components/com_guidedtours/config.xml:23]: It is possible to enter a float `6.5` which is a number but not a valid value in this case.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 00ac35d684188161f5e430048ef7b7b32542a7a3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44091">issues.joomla.org/tracker/joomla-cms/44091</a>.</sub>

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 00ac35d684188161f5e430048ef7b7b32542a7a3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44091">issues.joomla.org/tracker/joomla-cms/44091</a>.</sub>

**Comment by tecpromotion** [comment]
RTC

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44091">issues.joomla.org/tracker/joomla-cms/44091</a>.</sub>

**Comment by Hackwar** [comment]
Thank you for your contribution @obuisard!

**Pull request merged**